docker-compose.yml

```diff
@@ -37,8 +37,8 @@ services:
       # - ./docker/db/data:/var/lib/mysql:rw
   nginx:
     build:
-      context: ./docker/nginx
-      dockerfile: ./Dockerfile
+      context: ./
+      dockerfile: ./Dockerfile.nginx
```

Comment:
I don't think so, we need this one in order to do the ADD, we don't need all Dockerfile to have the app context